.github/workflows/trusted_partners.js

```diff
@@ -58,11 +58,22 @@ const filter_action = async ({github, context}) => {
   if (title && title.toLowerCase().includes("onednn"))
     assignees = onednn_assignees;
   const intel_windows_assignees = ['nitins17', 'learning-to-play'];
-  if (title && title.toLowerCase().includes("intel") && title.toLowerCase().includes("windows"))
+  if (title && title.toLowerCase().includes("intel") && title.toLowerCase().includes("windows")  && domain.includes("intel.com"))
     assignees = intel_windows_assignees;
   const apple_silicon_assignees = ['penpornk', 'nitins17'];
-  if (title && title.toLowerCase().includes("apple") && title.toLowerCase().includes("silicon"))
+  if (title && title.toLowerCase().includes("apple") && title.toLowerCase().includes("silicon") && domain.includes("apple.com"))
     assignees = apple_silicon_assignees;
+  if (title && title.toLowerCase().includes("nvidia") && domain.includes("nvidia.com")) {
+    if (title.toLowerCase().includes("jax")) {
+      assignees.push('hawkinsp', 'yashk2810');
```

Comment:
I can also help out here, please add 'skye'

Reply:
Great, added! Thank you for offering!